### PR TITLE
logic functions

### PR DESCRIPTION
**Pull request by aepaysinger**
still need to do tests

### CODE REVIEW

File: logic_fuctions.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+def get_fresh_board():
+    fill = [None for _ in range(10)]
+
+
+def get_next_board(squares, i, x_is_next):
+    if squares[i] == "X":
+        x_is_next = "O"
+    else:
+        x_is_next = "X"
+    return x_is_next
```
Comment (W-Ely) [inline logic_fuctions.py:28]: Can you push and link to the code this is supposed to port?  This function looks wrong.

Comment (aepaysinger) [reply logic_fuctions.py:28]: is it because x_is_next needs to be xIsNext?

Comment (W-Ely) [reply logic_fuctions.py:28]: Not that.  Could you get the changes we have made in the react app pushed to github.

Comment (aepaysinger) [reply logic_fuctions.py:28]: i just pushed them

Comment (W-Ely) [reply logic_fuctions.py:28]: The snake casing here is correct, but this function isn't doing what this https://github.com/aepaysinger/react-tic-tac-toe/pull/1/files#diff-8f16384a8b1758c444d336f8516112ea3b4322fd4ae66893c2edbeeae1ae18b5R23 function does.

File: logic_functions.py [hunk 2]
```diff
@@ -21,9 +21,9 @@ def get_fresh_board():
 
 
 
-def get_next_board(squares, i, x_is_next):
-    if squares[i] == "X":
-        x_is_next = "O"
+def get_next_board(squaresB, i, x_is_next):
```
Comment (W-Ely) [inline logic_functions.py]: Why did you change the squares to squaresB here?

File: logic_functions.py [hunk 3]
```diff
@@ -21,9 +21,9 @@ def get_fresh_board():
...
+    if x_is_next == "X":
```
Comment (W-Ely) [inline logic_functions.py]: I think you need to study the code you are porting from, what do you expect `x_is_next` to be here?

File: logic_functions.py (hunk 2 above)
Comment (aepaysinger) [reply logic_functions.py]: in the Board.js file it says getNextBpard takes squaresB.

File: logic_functions.py (hunk 3 above)
Comment (aepaysinger) [reply logic_functions.py]: so I was thinking that is was "X" or "O" but now that i relook at it I think that x_is_next is either True or False

File: logic_functions.py (hunk 2 above)
Comment (W-Ely) [reply logic_functions.py]: Does the name of a variable have anything to do with the name of the parameter in a function signature?

Comment (aepaysinger) [reply logic_functions.py]: ugh no lol my bad

**Comment by W-Ely** [comment]
This PR was not ever approved, and you merged it.  The reason it wasn't approved is the functions were not correct.  Your backend is now broken.
